src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -2546,7 +2558,11 @@ void writeZszSpsz(EnergyPlusData &state,
 
     print(outputFile, "\nPeak Vol Flow (m3/s)");
     for (int i = 1; i <= numSpacesOrZones; ++i) {
-        if (!zsEquipConfig(i).IsControlled) continue;
+        int zoneNum = i;
+        if (forSpaces) {
+            zoneNum = state.dataHeatBal->space(i).zoneNum;
+        }
+        if (!state.dataHeatBal->Zone(zoneNum).IsControlled) continue;
```

Comment:
Only if you have any other changes.

    int zoneNum = (forSpaces) ? state.dataHeatBal->space(i).zoneNum : i;

Reply:
Yes, of course. That construct still is not my first thought, or second, or ...

Reply:
To add on, the parentheses around `forSpaces` is not needed unless you need it for order of operations. (Ternary has a very low operation precedence, so most things will be evaluated before it is applied.)